ftw/theming/resources/scss/globals/variables.scss

```diff
@@ -218,56 +192,37 @@ $screen-XXS: "(min-width: #{$mobile-tiny})" !default;
     @content;
   }
 }
-/*
-  Deprecated Variables
-  Will be removed in next major version.
- */
 
+/* These variable are deprecated */
+
+// Taken from http://en.wikipedia.org/wiki/Display_resolution
+$desktop-WQHD-width:   2560px !default;
+$desktop-FHD-width:    1920px !default;
+$desktop-HDPLUS-width: 1600px !default;
+$desktop-HD-width:     1360px !default;
+$desktop-WXGA-width:   1280px !default;
+$desktop-XGA-width:    1024px !default;
+
+// Taken from http://www.ciop.com/common-resolutions-for-mobile-phones-and-tablets
+$mobile-large: 1280px !default;
+$mobile-medium: 1024px !default;
+$mobile-small: 800px !default;
+$mobile-smaller: 600px !default;
+$mobile-tiny: 480px !default;
+
+$screen-XL: "(min-width: #{$desktop-WQHD-width})" !default;
+$screen-L: "(min-width: #{$desktop-HDPLUS-width})" !default;
+$screen-M: "(min-width: #{$desktop-HD-width})" !default;
+$screen-S: "(min-width: #{$desktop-XGA-width})" !default;
+$screen-XS: "(min-width: #{$mobile-small})" !default;
+$screen-XXS: "(min-width: #{$mobile-tiny})" !default;
 
-$primary-color: #205C90 !default;
-$secondary-color: #75ad0a !default;
-
-$gray-base: #403e3d !default;
-$gray-darker: lighten($gray-base, 13.5%) !default;
-$gray-dark: lighten($gray-base, 20%) !default;
-$gray: lighten($gray-base, 33.5%) !default;
-$gray-light: lighten($gray-base, 46.7%) !default;
-$gray-lighter: lighten($gray-base, 70%) !default;
-
-$standalone-color: #fafafa !default;
-$context-color: #337ab7 !default;
-$success-color: #5cb85c !default;
-$warning-color: #f0ad4e !default;
-$danger-color: #d9534f !default;
-
-$page-bg-color: $gray-lighter !default;
-$content-bg-color: #fff !default;
-$text-color: $gray-dark !default;
-$light-text-color: $gray-light !default;
-$link-color: $primary-color !default;
-$link-font-weight: normal !default;
-$link-hover-color: $secondary-color !default;
-$link-focus-color: $secondary-color !default;
-$link-decoration: none !default;
-$link-hover-decoration: underline !default;
-$outline-color: change-color($primary-color, $lightness: 60%) !default;
-
-$button-standalone-color: $standalone-color !default;
-$button-context-color: $context-color !default;
-$button-destructive-color: $danger-color !default;
-$button-success-color: $success-color !default;
-$button-warning-color: $warning-color !default;
-$button-danger-color: $danger-color !default;
-
-$button-standalone-color: $color-secondary !default;
-$button-context-color: $color-default !default;
-$button-destructive-color: $color-danger !default;
-$button-success-color: $color-success !default;
-$button-warning-color: $color-warning !default;
-$button-danger-color: $color-danger !default;
-$button-disabled-color: #ddd !default;
-$button-text-light-color: white !default;
-$button-text-dark-color: black !default;
+@include declare-variables(
+  screen-XL,
+  screen-L,
```

Comment:
Since those variables are deprecated I'd remove the `declare-variables` block, having the effect that those variables no longer appear in the control panel.